nextcord/application_command.py

```diff
@@ -0,0 +1,97 @@
+from __future__ import annotations
+from typing import TYPE_CHECKING, Union, Optional, List
+from enum import IntEnum
+from .mixins import Hashable
+from . import utils
+
+if TYPE_CHECKING:
+    from .state import ConnectionState
+    from .guild import Guild
+
+
+class ApplicationCommandType(IntEnum):
+    CHAT_INPUT = 1
+    USER = 2
+    MESSAGE = 3
+
+
+class ApplicationCommandOptionType(IntEnum):
+    SUB_COMMAND = 1
+    SUB_COMMAND_GROUP = 2
+    STRING = 3
+    INTEGER = 4
+    BOOLEAN = 5
+    USER = 6
+    CHANNEL = 7
+    ROLE = 8
+    MENTIONABLE = 9
+    NUMBER = 10  # A double, AKA floating point, AKA decimal.
```

Comment:
These enums can go in [enums.py](https://github.com/nextcord/nextcord/blob/b644c29841ec878b77b6a38bdb6e754bf7ec90e2/nextcord/enums.py) like the others.

Try to keep the style consistent with existing stuff as well (eg. lowercase, inherit `Enum`)